src/ProtectAgainstSpam.php

```diff
@@ -19,7 +19,7 @@ public function __construct(SpamResponder $spamResponder)
 
     public function handle(Request $request, Closure $next): Response
     {
-        if (! config('honeypot.enabled')) {
+        if (! config('honeypot.enabled') || $request->method() === 'GET') {
```

Comment:
If it's only on `POST` request, should this be `$request->method() !== 'POST'` instead?

Reply:
Yup